From c8af1ceccfa1c69845a0afb0068917cb16a4ef5b Mon Sep 17 00:00:00 2001
From: Joey Hewitt <joey@joeyhewitt.com>
Date: Sat, 2 Apr 2018 17:38:00 -0700
Subject: [PATCH] point storage to udev-linked paths

udev rules in this package create symlinks the daemon opens
---
 storage.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/storage.c b/storage.c
index 997b428..8754b98 100644
--- a/storage.c
+++ b/storage.c
@@ -23,10 +23,10 @@ struct caller {
 };
 
 static const struct partition partition_table[] = {
-	{ "/boot/modem_fs1", "/boot/modem_fs1" },
-	{ "/boot/modem_fs2", "/boot/modem_fs2" },
-	{ "/boot/modem_fsc", "/boot/modem_fsc" },
-	{ "/boot/modem_fsg", "/boot/modem_fsg" },
+	{ "/boot/modem_fs1", "/dev/disk/qcom_rmtfs/modem_fs1" },
+	{ "/boot/modem_fs2", "/dev/disk/qcom_rmtfs/modem_fs2" },
+	{ "/boot/modem_fsc", "/dev/disk/qcom_rmtfs/modem_fsc" },
+	{ "/boot/modem_fsg", "/dev/disk/qcom_rmtfs/modem_fsg" },
 	{}
 };
 
-- 
2.7.4

